cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -12,6 +12,8 @@ metadata:
   name: rook-ceph
 allowPrivilegedContainer: true
 allowHostDirVolumePlugin: true
+allowHostNetwork: false
```

Comment:
@leseb we are setting these values to `true` for csi SCC. Do we need to make `true` here too?

Reply:
Rook SCC is not the same as csi SCC. csi pods should run on hostnetworking that's why we always set that to true. just a question from my side what if someone wants to run rook with the provider as host in openshift down we need to set this to true? Hope it's documented somewhere.

Reply:
@Madhu-1 yes than this should be documented because this setting can be missed easily.

How about this [document](https://github.com/rook/rook/blob/master/Documentation/ceph-advanced-configuration.md#use-hostnetwork-in-the-rook-ceph-cluster-configuration) where we explain about the host provider, we can also explain how to use it for Openshift and can explain this field modification?